src/model/AirLoopHVACDedicatedOutdoorAirSystem.cpp

```diff
@@ -378,6 +385,10 @@ namespace model {
     getImpl<detail::AirLoopHVACDedicatedOutdoorAirSystem_Impl>()->removeAllAirLoops();
   }
 
+  std::vector<IdfObject> AirLoopHVACDedicatedOutdoorAirSystem::remove() {
+    return getImpl<detail::AirLoopHVACDedicatedOutdoorAirSystem_Impl>()->remove();
+  }
+
```

Comment:
```suggestion
```

The public side remove() is already defined in WorkspaceObject and forwards to the Impl, which is virtual, so no need to define it in the public class.